featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -746,6 +749,53 @@ def test_makes_direct_features_through_multiple_relationships(games_es):
                 assert feature_with_name(features, f)
 
 
+def test_stacks_multioutput_features(es):
+    class TestTime(TransformPrimitive):
+        name = "test_time"
+        input_types = [Datetime]
+        return_type = Numeric
+        number_output_features = 6
+
+        def get_function(self):
+            def test_f(x):
+                times = pd.Series(x)
+                units = ["year", "month", "day", "hour", "minute", "second"]
+                return [times.apply(lambda x: getattr(x, unit)) for unit in units]
+            return test_f
+
+    fm, feat = ft.dfs(entityset=es,
```

Comment:
we can skip calculating the feature matrix by using `features_only` option since we don't actually use matrix